src/app/components/mnoe-staff-dashboards-list/mnoe-staff-dashboards-list.coffee

```diff
@@ -89,8 +89,6 @@
     # Advisor Dashboard
     # -----------------------------------------------------------
     vm.createDashboard = (dashboard) ->
-      angular.merge(dashboard, { organization_ids: [vm.organization.id] })
```

Comment:
I assume this deletion is to mirror changes you made on the backend?

Reply:
I realised this wasn't actually needed for single company dashboard but it breaks the multiple company dashboard as it overrides the `organization_ids`